linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/oval/shared.xml

```diff
@@ -28,4 +33,35 @@
   </ind:textfilecontent54_state>
 
   <external_variable comment="external variable for the desired value of Compression" datatype="string" id="var_sshd_disable_compression" version="1" />
+
+  <ind:variable_test check="all" check_existence="all_exist"
+  comment="tests if the variable is selecting default value (delayed)"
+  id="test_var_sshd_disable_compression_is_default" version="1">
+    <ind:object object_ref="object_var_sshd_disable_compression" />
+    <ind:state state_ref="state_sshd_disable_compression_default_value" />
+  </ind:variable_test>
+
+  <local_variable id="sshd_disable_compression_default_value" datatype="string" version="1" comment="Delayed">
+    <literal_component>delayed</literal_component>
+  </local_variable>
+
+  <ind:variable_object comment="All modified times of all keyfiles" id="object_var_sshd_disable_compression" version="1">
+     <ind:var_ref>var_sshd_disable_compression</ind:var_ref>
+   </ind:variable_object>
+
+  <ind:variable_state id="state_sshd_disable_compression_default_value" version="1">
+    <ind:value datatype="int" operation="equals" var_check="all" var_ref="sshd_disable_compression_default_value" />
+  </ind:variable_state>
+
+  <ind:textfilecontent54_test check="all" check_existence="none_exist"
+  comment="tests absence of the value of Compression setting in the /etc/ssh/sshd_config file"
+  id="test_absence_of_sshd_disable_compression" version="1">
+    <ind:object object_ref="obj_absence_of_sshd_disable_compression" />
```

Comment:
I think you can reuse the same object defined by the macro `oval_line_in_file_object` above.

Reply:
You are right.